graylog-storage-opensearch2/src/main/java/org/graylog/storage/opensearch2/MoreSearchAdapterOS2.java

```diff
@@ -62,7 +61,7 @@
 
 public class MoreSearchAdapterOS2 implements MoreSearchAdapter {
     private static final Logger LOG = LoggerFactory.getLogger(MoreSearchAdapterOS2.class);
-    public static final IndicesOptions INDICES_OPTIONS = IndicesOptions.fromOptions(false, false, true, false);
+    public static final IndicesOptions INDICES_OPTIONS = IndicesOptions.fromOptions(true, true, true, false);
```

Comment:
The same as here: https://github.com/Graylog2/graylog2-server/pull/18146/files#r1479887434

Reply:
done